source/particle/property/initial_composition.cc

```diff
@@ -53,7 +44,7 @@ namespace aspect
         for (unsigned int i = 0; i < this->n_compositional_fields(); i++)
           {
             std::ostringstream field_name;
-            field_name << "initial C_" << i;
+            field_name << "initial composition_" << i;
```

Comment:
Yes, good point. Should have done that in the original implementation already. Will do.